config/options

```diff
@@ -4,77 +4,73 @@ if [[ $EUID -eq 0 ]]; then
 
 # determines DISTRO, if not forced by user
 # default is LibreELEC
-if [ -z "$DISTRO" ]; then
-  DISTRO="LibreELEC"
-else
-  DISTRO="$DISTRO"
-fi
+DISTRO="${DISTRO:-LibreELEC}"
 
 # determines PROJECT, if not forced by user
 # default is Generic
-if [ -z "$PROJECT" ]; then
-  PROJECT="Generic"
-else
-  PROJECT="$PROJECT"
-fi
+PROJECT="${PROJECT:-Generic}"
 
-# determines TARGET_ARCH, if not forced by user (x86_64 / arm)
+# determines TARGET_ARCH, if not forced by user
 # default is x86_64
-if [ -z "$ARCH" ]; then
-  TARGET_ARCH="x86_64"
-else
-  TARGET_ARCH="$ARCH"
-fi
+ARCH="${ARCH:-x86_64}"
+TARGET_ARCH="${ARCH}"
 
 # include helper functions
-. config/functions
+  . config/functions
 
 # include versioning
   . config/version
 
-# read distro versioning if available
- if [ -f "$DISTRO_DIR/$DISTRO/version" ]; then
-   . $DISTRO_DIR/$DISTRO/version
- fi
+# read DISTRO versioning if available
+  if [ -f "${DISTRO_DIR}/${DISTRO}/version" ]; then
+    . ${DISTRO_DIR}/${DISTRO}/version
+  fi
 
-# read distro options if available
- if [ -f "$DISTRO_DIR/$DISTRO/options" ]; then
-   . $DISTRO_DIR/$DISTRO/options
- fi
+# read DISTRO options if available
+  if [ -f "${DISTRO_DIR}/${DISTRO}/options" ]; then
+    . ${DISTRO_DIR}/${DISTRO}/options
+  fi
 
-# read project options if available
- if [ -f "$PROJECT_DIR/$PROJECT/options" ]; then
-   . $PROJECT_DIR/$PROJECT/options
- fi
+# read PROJECT options if available
+  if [ -f "${PROJECT_DIR}/${PROJECT}/options" ]; then
+    . ${PROJECT_DIR}/${PROJECT}/options
+  fi
 
-# read board options if available
- if [ -f "$PROJECT_DIR/$PROJECT/devices/$DEVICE/options" ]; then
-   . $PROJECT_DIR/$PROJECT/devices/$DEVICE/options
+# read DEVICE options if available
+ if [ -f "${PROJECT_DIR}/${PROJECT}/devices/${DEVICE}/options" ]; then
+   . ${PROJECT_DIR}/${PROJECT}/devices/${DEVICE}/options
  fi
 
 # projects can set KERNEL_NAME (kernel.img)
-[ -z "$KERNEL_NAME" ] && KERNEL_NAME="KERNEL"
+KERNEL_NAME="${KERNEL_NAME:-KERNEL}"
+
+LINUX_DEPENDS="${PROJECT_DIR}/${PROJECT}/linux ${PROJECT_DIR}/${PROJECT}/patches/linux \
+  ${PROJECT_DIR}/${PROJECT}/packages/linux \
+  ${ROOT}/packages/linux"
 
-LINUX_DEPENDS="$PROJECT_DIR/$PROJECT/linux $PROJECT_DIR/$PROJECT/patches/linux $PROJECT_DIR/$PROJECT/packages/linux $ROOT/packages/linux"
-[ -n "$DEVICE" ] && LINUX_DEPENDS+=" $PROJECT_DIR/$PROJECT/devices/$DEVICE/linux $PROJECT_DIR/$PROJECT/devices/$DEVICE/patches/linux $PROJECT_DIR/$PROJECT/devices/$DEVICE/packages/linux"
-[ "$TARGET_ARCH" = "x86_64" ] && LINUX_DEPENDS+=" $ROOT/packages/linux-firmware/intel-ucode $ROOT/packages/linux-firmware/kernel-firmware"
+[ -n "${DEVICE}" ] && LINUX_DEPENDS+=" ${PROJECT_DIR}/${PROJECT}/devices/${DEVICE}/linux \
+  ${PROJECT_DIR}/${PROJECT}/devices/${DEVICE}/patches/linux \
+  ${PROJECT_DIR}/$PROJECT/devices/${DEVICE}/packages/linux"
+
+[ "${TARGET_ARCH}" = "x86_64" ] && LINUX_DEPENDS+=" ${ROOT}/packages/linux-firmware/intel-ucode \
+  ${ROOT}/packages/linux-firmware/kernel-firmware"
```

Comment:
I might be in the minority but I preferred a longer, single line for stuff like this.